proto/sentry_protos/snuba/v1/trace_item_filter.proto

```diff
@@ -50,3 +50,35 @@ message TraceItemFilter {
     ExistsFilter exists_filter = 5;
   }
 }
+
+// Wraps a TraceItemFilter and specifies the trace item name,
+// to be used for cross-event queries.
+message EventFilter {
+  string trace_item_name = 1;
```

Comment:
What are all the possible options for `trace_item_name`? From the examples, I see we have `spans` and `errors`. Are there more? Can this be an enum?

Reply:
Ultimately it would be one for each event type that can be connected to a trace (replays, profiles, functions).

Having an Enum makes sense, we can change that.

@volokluev Can I change that in v1 or should I wait until v2?

Reply:
There already is an enum:

https://github.com/getsentry/sentry-protos/blob/main/proto/sentry_protos/snuba/v1/request_common.proto#L19